composer.json

```diff
@@ -20,7 +20,8 @@
 		"symfony/form": "~2.7|~3.0",
 		"symfony/http-kernel": "~2.7|~3.0",
 		"symfony/translation": "~2.7|~3.0",
-		"symfony/yaml": "~2.7|~3.0"
+		"symfony/yaml": "~2.7|~3.0",
+		"paragonie/random_compat": "^1|^2"
```

Comment:
To let tests pass with Symfony 2.7, the dependency for `paragonie/random_compat` must be added after the Symfony ones. Otherwise, installing the latest version of `paragonie/random_compat` has higher priority than installing the latest version of Symfony. Thus, Symfony 2.7.8 will be used instead of the latest 2.7.x version since Symfony > 2.7.8 explicitly requires `paragonie/random_compat` 1.0 and cannot be used with 2.0. Symfony > 2.8 is not affected by this issue.